### PR TITLE
fix-windows issues

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: azure-pipelines/ci.yml [hunk 1]
```diff
@@ -47,8 +55,16 @@ jobs:
...
           
   - job: TestReport
+    strategy:
+      matrix:
+        linux:
+          imageName: 'ubuntu-20.04'
+        mac:
+          imageName: 'macos-10.14'
+        windows:
+          imageName: 'windows-2019'
```
Comment (lirenhe) [inline azure-pipelines/ci.yml:65]: can we use windows-latest?

File: azure-pipelines/ci.yml [hunk 2]
```diff
@@ -69,8 +85,17 @@ jobs:
...
      
   - job: Analyze
+    strategy:
+      matrix:
+        linux:
+          imageName: 'ubuntu-20.04'
+        mac:
+          imageName: 'macos-10.14'
+        windows:
+          imageName: 'windows-2019'
```
Comment (lirenhe) [inline azure-pipelines/ci.yml:95]: Same above

File: azure-pipelines/ci.yml (hunk 1 above)
Comment (qiaozha) [reply azure-pipelines/ci.yml:65]: sure

File: test/scenario-test.ts [hunk 3]
```diff
@@ -54,14 +54,24 @@ export class Process {
         for (const k in extraOption) {
             cmdOption.push('--' + k + '=' + extraOption[k]);
         }
-        const cmd =
+        let cmd =
             path.join(`${__dirname}`, '/../../' + 'node_modules/.bin/autorest') +
             ' --version=3.0.6320 --az --use=' +
```
Comment (lirenhe) [inline test/scenario-test.ts:59]: will you also update this version?

Comment (qiaozha) [reply test/scenario-test.ts:59]: This is autorest core version. I think we'd better not change it

**Review by lirenhe — APPROVED** [review]

**Pull request merged**